### PR TITLE
More ship names!

### PR DESCRIPTION
**Pull request by Pointedstick**
You know the drill.

### CODE REVIEW

File: data/names.txt [hunk 1]
```diff
@@ -1029,6 +1060,19 @@ phrase "standalone civilian ship names"
 		"Tiamat"
 		"Artemis"
 		"Annabel Lee"
+		"Old Ironsides"
+		"20 Chanri"
```
Comment (Bladewood) [inline data/names.txt]: We sure putting digits in (human) ship names is okay?

Comment (Pointedstick) [reply data/names.txt]: Why wouldn't it be?

Comment (Bladewood) [reply data/names.txt]: Just that no other human ship name follows that pattern, as far as I know. They all use the full word forms, not digit forms.

Comment (KiLEdEnNis) [reply data/names.txt]: it feels more [robot-like? non-personal?] to me, but a few of them shouldn't be a problem

**Pull request merged**